components/brave_wallet_ui/components/desktop/views/crypto/index.tsx

```diff
@@ -150,6 +159,8 @@ export const CryptoView = ({ sessionRoute }: Props) => {
     }
   }, [location.key])
 
+  const { pageHandler } = getWalletPageApiProxy()
```

Comment:
please move this to somewhere above `// methods` so that it is declared before use